### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by sneaks** [akalajzi/atom-file-bookmark#6]
[Enter steps to reproduce below:]
i just book marked a folder
1. ...
2. ...

**Atom Version**: 1.5.4
**System**: Mac OS X 10.11.3
**Thrown From**: [file-bookmark](https://github.com/akalajzi/atom-file-bookmark) package, v0.3.1
### Stack Trace

Uncaught TypeError: Cannot read property 'split' of undefined

```
At /Users/snx/.atom/packages/file-bookmark/lib/file-bookmark-view.coffee:169

TypeError: Cannot read property 'split' of undefined
    at FileBookmarkView.module.exports.FileBookmarkView._splitPathnameAndFilename (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark-view.coffee:169:25)
    at FileBookmarkView.module.exports.FileBookmarkView._groupPaths (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark-view.coffee:154:15)
    at FileBookmarkView.module.exports.FileBookmarkView.renderItems (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark-view.coffee:87:21)
    at FileBookmarkView.module.exports.FileBookmarkView.redrawBookmarks (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark-view.coffee:116:6)
    at FileBookmark.add (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark.coffee:120:25)
    at FileBookmark.toggleBookmark (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark.coffee:112:8)
    at HTMLSpanElement.<anonymous> (/Users/snx/.atom/packages/file-bookmark/lib/file-bookmark.coffee:64:68)
    at HTMLDivElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```
### Commands

```
     -0:27.5.0 project-manager:list-projects (atom-text-editor.editor.is-focused)
  2x -0:05.5.0 tree-view:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```
### Config

``` json
{
  "core": {
    "audioBeep": false,
    "disabledPackages": [
      "flex-tool-bar"
    ],
    "ignoredNames": [
      ".DS_Store",
      ".git",
      ".codekit"
    ],
    "projectHome": "/Users/Snx/Dropbox/Sites"
  }
}
```
### Installed Packages

``` coffee
# User
atom-silverstripe, v1.12.0
emmet, v2.4.2
file-bookmark, v0.3.1
minimap, v4.21.0
pigments, v0.24.5
project-manager, v2.7.6
tool-bar, v0.3.0
tool-bar-atom, v1.2.0
tool-bar-main, v0.0.9

# Dev
No dev packages
```

**Comment by Samuel29** [comment]
Hi
after a fresh install of the plugin; I wanted to bookmark a file for the 1st time, then got that error :/

**Comment by akalajzi** [comment]
Though I wasnt able to reproduce this, it should be fixed in 0.4.0 version. I hope at least.

How did you try to bookmark a folder @sneaks ?
